lib/cdo/aws/cloudfront.rb

```diff
@@ -151,7 +151,14 @@ def self.distribution_config(app, origin, aliases, ssl_cert=nil)
               ErrorCode: error,
               ResponseCode: error,
               ResponsePagePath: '/assets/error-pages/site-down.html'
-            }
+            }.tap do |error_response_hash|
+              # Don't use friendly error pages on LevelBuilder to ensure content editors can debug errors.
+              if CDO.pegasus_hostname == 'levelbuilder.code.org'
+                error_response_hash[:ErrorCachingMinTTL] = 0
+                error_response_hash.delete(:ResponseCode)
+                error_response_hash.delete(:ResponsePagePath)
```

Comment:
it's hard to tell from the docs, but do you also need to `.delete(:ErrorCode)`?

Reply:
No, we want to preserve the `:ErrorCode` key in this case.